components/ws-manager/pkg/manager/create.go

```diff
@@ -501,6 +501,11 @@ func (m *Manager) createDefiniteWorkspacePod(startContext *startWorkspaceContext
 		},
 	}
 
+	var terminationGracePeriodSeconds *int64
+	if req.Spec.TerminationGracePeriodSeconds != 0 {
```

Comment:
No upper limit? Even for free accounts?

Reply:
👍 It makes sense to set an upper limit. Any suggestion for what is reasonable?
I wouldn't want to bind it to the account type, though.
We might want to reconsider measurement of credits , though, which is currently based on `stoppingTime`. cc @easyCZ

Reply:
In this context the upper bound would only need to be sensible from an operations perspective, e.g. to not trip adverse behaviour or alerts.

For now, we could hardcode it, and if future need arrises plug it through config.

Reply:
Let's add 5 min limit? Feels like it might be a good starting point?